### ISSUE TITLE
[Snackbar] Messages with Actions dismiss automatically when Switch Control is in use

### ISSUE THREAD

**Issue by romoore** [material-components/material-components-ios#4713]
Users of a Switch Control cannot access Snackbar Messages presented either with or without actions.

## Reproduction steps
1. Turn on Switch Control
1. Open the Catalog &rarr; Snackbar &rarr; Start Demo
1. Activate "Snackbar  with Action Button"

## Expected behavior
The user's focus is taken to the Snackbar message so they may choose to activate the action.

## Actual behavior
The user's focus is taken back to the "Back" button but the user cannot access to the Snackbar message in any fashion. 

## Platform (please complete the following information)
iPhone 5c/iOS 10.3.3

## Additional context
VoiceOver will correctly change focus to the SnackbarMessage, but Switch Control will not.

**Comment by romoore** [comment]
Let's post to our internal iOS development community to see if anyone has a good idea for a way forward on this issue.
